package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "prismic-toolbar",
-  "version": "4.0.4",
+  "version": "4.0.5",
```

Comment:
Not sure if I'm supposed to change this version manually. Also there is a different version in `package.lock`
https://github.com/prismicio/prismic-toolbar/blob/7211967745c285ab77c2686f25e5ef80418507bf/package-lock.json#L3